src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -724,7 +736,8 @@ def update_connectedk8s(cmd, instance, tags=None):
 
 
 def update_agents(cmd, client, resource_group_name, cluster_name, https_proxy="", http_proxy="", no_proxy="", proxy_cert="",
-                  disable_proxy=False, kube_config=None, kube_context=None, auto_upgrade=None):
+                  disable_proxy=False, kube_config=None, kube_context=None, auto_upgrade=None, container_registry_repository="",
+                  container_registry_username="", container_registry_password=""):
```

Comment:
What would be the behavior in case some user does not pass these parametes in CLI but instead sets them in the values file?

Reply:
It will same as other values, values defined in the value file will be applied to the helm install/upgrade